troubleshooting.md

```diff
@@ -143,3 +143,23 @@ If you are using a useradd command within a Dockerfile with a large UID/GID, it
 #### Solution
 
 If the entry in the Dockerfile looked like: RUN useradd -u 99999000 -g users newuser then add the `--log-no-init` parameter to change it to: `RUN useradd --log-no-init -u 99999000 -g users newuser`. This option tells useradd to stop creating the lastlog file.
+
```

Comment:
Need to add the line break, title and short description.  Perhaps:
```
---
### 7) Permission denied when running Podman commands
When the container doesn't have permission to mount the storage directory for the container a permission error will be raised.
```

Reply:
Fixed